app/models/application_reference.rb

```diff
@@ -136,4 +136,10 @@ def feedback_overdue?
 
     replace_referee_at < Time.zone.now
   end
+
+  def provider_name
```

Comment:
I am not happy with this name. Let me know if you have any suggestions!

Reply:
i think this name is ok! I can't really think of what else to call it apart from `name_of_the_provider` which seems to just be a verbose way of saying `provider_name` 🤷‍♂️